8Knot/pages/repo_overview/visualizations/repo_general_info.py

```diff
@@ -31,6 +32,7 @@
                 dcc.Loading(
                     html.Div(id=f"{PAGE}-{VIZ_ID}"),
                 ),
+                dbc.Row([dbc.Label(["Last Updated:", html.Span(id=f"{PAGE}-{VIZ_ID}-updated")], className="mr-2")]),
```

Comment:
im assuming this is from the linter? this is so strange, it used to not do this

Reply:
yep. i wonder if black has updated or something.

Reply:
that would make sense